src/services/apiCall.js

```diff
@@ -0,0 +1,23 @@
+
+// function apiCall(url, method, json) {
+function apiCall(url, method, json) {
+  if(method === 'get' || method === 'post' || method === 'put' || method === 'delete') {
+    return fetch(url, { method }) 
+      .then(res => res.json())
+      .catch(err => {
+        console.log(err);
+      });
+  } else {
```

Comment:
not sure that you need this whole part. i think you could just pass the method and json into the fetch and you should be fine.